mozci/data/contract.py

```diff
@@ -48,6 +48,7 @@ class Contract:
                         "exception",
                         "canceled",
                         "superseded",
+                        "unknown",
```

Comment:
In other data sources, we return a task object without a `result` if the task hasn't started yet or is still running, we should do the same for the Treeherder data source

Reply:
Done